tuta-sdk/rust/src/date.rs

```diff
@@ -0,0 +1,56 @@
+use std::fmt::Formatter;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use serde::de::{Error, Visitor};
+use crate::UniffiCustomTypeConverter;
+
+#[derive(Copy, Clone, Default, PartialEq, PartialOrd)]
+#[repr(transparent)]
+pub struct Date(i64);
+
+impl Date {
```

Comment:
I think the other Date things were converted into native date things for Kotlin/Swift… might be annoying with a custom one